src/components/Builder/Achievement/index.jsx

```diff
@@ -92,10 +93,46 @@ const Achievement = ({ achievementData }) => {
     }
   };
 
-  const onReset = () => {
-    form.resetFields();
+  const handleUpdate = async (values) => {
+    try {
+      for (const achievement of values) {
+        if (achievement.id) {
+          const response = await updateAchievementService({
+            profile_id: profile_id,
+            achievement_id: achievement.id,
+            values: achievement,
+          });
+          if (response.data?.message) {
+            toast.success(response.data?.message);
```

Comment:
why we need toast here ? interceptor toaster will not help here ?

Reply:
As per discussion, kept as it is